lib/ui/window/window.h

```diff
@@ -20,18 +20,24 @@
 namespace flutter {
 class Window final {
  public:
-  Window(int64_t window_id, ViewportMetrics metrics);
+  Window(tonic::DartPersistentValue& library,
+         int64_t window_id,
+         ViewportMetrics metrics);
 
   ~Window();
 
   int window_id() const { return window_id_; }
 
   const ViewportMetrics& viewport_metrics() const { return viewport_metrics_; }
 
+  // Send an AddView message to Dart with the current view metrics.
```

Comment:
Consider mentioning that it is an error to add a view with the same `id` multiple times.

```suggestion
  // Send an AddView message to Dart with the current view metrics.
  // This must not be called more than once per `window_id`.
```